src/components/Wizard/WizardStyles.ts

```diff
@@ -10,3 +10,12 @@ export const CancelButtonStyle = styled(Button)`
 	align-self: center;
 	padding-left: 10px;
 `;
+
+export const ModalContentStyle = styled.div`
+	padding: ${themeSpacing(8)};
+`;
+
+export const ModalTopBarStyle = styled.h1`
+	font-weight: 700;
+	font-size: 18px;
+`;
```

Comment:
Biedt de ASC UI CL zelf out of the box geen header component?
en als je het op deze manier doet, moet een font size dan ook niet uit de CL komen, net als de themeSpacing?

Reply:
I've adjusted this to (Heading coming from asc-ui defaulting to H1):
```
export const ModalTopBarStyle = styled(Heading)`
	font-size: 18px;
`;
```

Reply:
Lijkt me niet de juiste wijze. Want een change in ASC-UI in h1 fontsize gaat dus niet automatisch hierin mee, toch?

Reply:
That's by design. Semantically we need a H1. Font-family/weight-wise as well. But size in a modal heading has been set to a different value :man_shrugging: